### PR TITLE
added optional `initialTab` .. debug layer config option

### PR DESCRIPTION
**Pull request by ycw**
``` 
scene.debugLayer.show({
  initialTab: "statistics"
});
```

`initialTab` is one of 
- `DebugLayer.ACTION_TAB.PROPERTIES`
- `DebugLayer.ACTION_TAB.DEBUG`
- `DebugLayer.ACTION_TAB.TOOLS`
- `DebugLayer.ACTION_TAB.STATISTICS`
- `DebugLayer.ACTION_TAB.SETTINGS`

corresponding string values are 
- `"properties"`
- `"debug"`
- `"tools"`
- `"statistics"`
- `"settings"`

default is `"properties"`

### CODE REVIEW

File: inspector/src/components/actionTabs/actionTabsComponent.tsx [hunk 1]
```diff
@@ -24,9 +24,20 @@ interface IActionTabsComponentProps {
     popupMode?: boolean,
     onPopup?: () => void,
     onClose?: () => void,
-    globalState?: GlobalState
+    globalState?: GlobalState,
+    initialTab?: string
```
Comment (sebavan) [inline inspector/src/components/actionTabs/actionTabsComponent.tsx]: Why not relying on an enum ?

**Comment by sebavan** [comment]
I wonder if we should not use an enum instead to ensure only the available config are supported.

Asking @deltakosh  For review ?

File: inspector/src/components/actionTabs/actionTabsComponent.tsx (hunk 1 above)
Comment (ycw) [reply inspector/src/components/actionTabs/actionTabsComponent.tsx]: i found that /src/tsconfig.json restricted 'rootDir' to './', implied that there's no way to 
> export an `enum` from /inspector/src/..

and then 

> import it to /src/Debug/debugLayer.ts 

so i chose to use `string`

File: src/Debug/debugLayer.ts [hunk 2]
```diff
@@ -121,6 +125,17 @@ export class DebugLayer {
      */
     public static InspectorURL = `https://unpkg.com/babylonjs-inspector@${Engine.Version}/babylon.inspector.bundle.js`;
 
+    /**
+     * Action tab kinds
+     */
+    public static ACTION_TAB = {
```
Comment (deltakosh) [inline src/Debug/debugLayer.ts]: why not exporting this as an enum (export enum DebugLayerTabs...) so you do not need to recreate it in the action tab class

Comment (ycw) [reply src/Debug/debugLayer.ts]: do you mean that export enum from debugLayer.ts, then inspector.ts import that enum?

Comment (ycw) [reply src/Debug/debugLayer.ts]: > 
> 
> why not exporting this as an enum (export enum DebugLayerTabs...) so you do not need to recreate it in the action tab class

imo, the debuglayer is a consumer of inspector.. what tabs are available in debuglayer should be the tabs exporting from inspector.. should not be opposite, what do you think?

Comment (ycw) [reply src/Debug/debugLayer.ts]: > 
> 
> > why not exporting this as an enum (export enum DebugLayerTabs...) so you do not need to recreate it in the action tab class
> 
> imo, the debuglayer is a consumer of inspector.. what tabs are available in debuglayer should be the tabs exporting from inspector.. should not be opposite, what do you think?

the reason why i can't do that has mentioned at https://github.com/BabylonJS/Babylon.js/pull/7235#discussion_r352292106 :D

Comment (deltakosh) [reply src/Debug/debugLayer.ts]: Yes this is weird I agree but we are already doing it. It is ok to expose it and import it from the inspector

Comment (ycw) [reply src/Debug/debugLayer.ts]: ok.

**Comment by ycw** [comment]
usage

```
scene.debugLayer.show({
  initialTab: DebugLayerTab.STATISTICS
});
```

`initialTab` options:
- `DebugLayerTab.PROPERTIES` (default)
- `DebugLayerTab.DEBUG`
- `DebugLayerTab.STATISTICS`
- `DebugLayerTab.TOOLS`
- `DebugLayerTab.SETTINGS`

File: src/Debug/debugLayer.ts [hunk 3]
```diff
@@ -108,6 +112,32 @@ Object.defineProperty(Scene.prototype, "debugLayer", {
...
 });
 
+/**
+ * Enum of inspector action tab
+ */
+export enum DebugLayerTab {
+    /**
+     * Properties tag (default)
+     */
+    PROPERTIES = 0,
```
Comment (deltakosh) [inline src/Debug/debugLayer.ts]: Enums are not upper case:
https://github.com/BabylonJS/Babylon.js/blob/a407a04631d848ac89535d64282af004f0ec2cea/src/Materials/Node/Enums/nodeMaterialSystemValues.ts#L4

Comment (deltakosh) [reply src/Debug/debugLayer.ts]: But not a big deal..I will fix that after the merge ;)

Thanks a ton!

**Pull request merged**